src/components/dialogs/parameters/common/limitreductions/limit-reductions-table-form.tsx

```diff
@@ -54,30 +48,6 @@ const LimitReductionsTableForm: FunctionComponent<{
         return columnsDefinition;
     }, [intl, limits, getLabelColumn]);
 
-    const toFormValues = useCallback(() => {
-        return {
-            [LIMIT_REDUCTIONS_FORM]: limits.map((vlLimits) => {
-                return {
-                    [VOLTAGE_LEVELS_FORM]: vlLimits.voltageLevel.nominalV + ' (kV)',
-                    [IST_FORM]: vlLimits.permanentLimitReduction,
-                    ...toFormValuesFromTemporaryLimits(vlLimits.temporaryLimitReductions),
-                };
-            }),
-        };
-    }, [limits]);
-
-    const toFormValuesFromTemporaryLimits = (limits: ITemporaryLimitReduction[]) => {
-        let formValues: Record<string, number> = {};
-        limits.forEach((limit, index) => {
-            formValues[LIMIT_DURATION_FORM + index] = limit.reduction;
-        });
-        return formValues;
-    };
-
-    useEffect(() => {
-        reset(toFormValues());
-    }, [reset, toFormValues]);
-
```

Comment:
added back only in SA ?

Reply:
you are right.
i fixed it by adding a new custuom hook to reuse this logic . useLimitReductionsForm.ts

Reply:
If it's to use it in LF and SA, I'm not sure why we need to move it and not keep it here?